types/nodemailer-mailgun-transport/index.d.ts

```diff
@@ -15,6 +15,11 @@ declare namespace mailgunTransport {
         domain?: string;
     }
 
+    interface AliasAuthOptions {
+        apiKey: string;
+        domain?: string;
+    }
+
     interface Options {
         auth: AuthOptions;
```

Comment:
Wouldn’t it be simpler to just make this `auth: AuthOptions | AliasAuthOptions`?

Reply:
thanks for the reply. Honestly, I just wanted to define alias type to make the auto correct does not show `api_key` if we already provide `apiKey`, but I didn't know how to do it and forgot to change the code like your solution.

Let me fix this